android-lib/src/main/java/com/kinvey/android/store/UserStore.java

```diff
@@ -807,14 +807,28 @@ public static void loginWithAuthorizationCodeAPI(@NonNull AbstractClient client,
      * @param password {@link String} the password of Kinvey user.
      * @param redirectURI redirectURI
      * @param callback {@link KinveyUserCallback}
-     * @deprecated Use {@link #loginWithMIC(AbstractClient, String, String, String, String, KinveyUserCallback)}
+     * @deprecated Use {@link #loginWithMIC(AbstractClient, String, String, String, KinveyUserCallback)}
      */
     public static void loginWithAuthorizationCodeAPI(@NonNull AbstractClient client, @NonNull String username,
                                                      @NonNull String password, @Nullable String clientId,
                                                      @NonNull String redirectURI, @NonNull KinveyUserCallback<User> callback){
         loginWithMIC(client, username, password, clientId,redirectURI, callback);
     }
 
+    /***
+     *
+     * Login with the MIC service, using the oauth flow.  This method provides direct login, without rending a login page and without redirectURI.
```

Comment:
This first sentence should read "Login with the MIC service, using the resource owner grant flow."

Reply:
Ok, fixed